CHANGELOG.md

```diff
@@ -1,3 +1,57 @@
+## [15.5.0](https://github.com/commercetools/merchant-center-application-kit/compare/v15.4.2...v15.5.0) (2019-12-09)
+
+#### ⛑ Type: Refactoring
+
+- `application-components`, `application-shell`, `i18n`, `jest-stylelint-runner`, `mc-scripts`, `react-notifications`
+
+  - [#1193](https://github.com/commercetools/merchant-center-application-kit/pull/1193) refactor: use uikit single packages ([@emmenko](https://github.com/emmenko))
+
+  Packages that previously required `@commercetools-frontend/ui-kit` as a peer dependency now require peer dependencies of specific uikit packages.
+
+  > If you keep using `@commercetools-frontend/ui-kit` you don't need to change anything. If you start using specific uikit packages though, you need to make sure you have the proper peer dependencies in place.
+
+  - `application-components`
```

Comment:
Would it be worht linking to `install-peerdeps` to install those peer deps in that case automatically?

Reply:
Yeah I think we should add this to each package README also.